src/main/java/net/sf/jabref/external/ExternalFileTypeEntryEditor.java

```diff
@@ -184,10 +181,16 @@ public void changedUpdate(DocumentEvent documentEvent) {
             });
         }
 
+        String title = "Edit file type";
+        
+        if(entry.getName().isEmpty()) {
+            title = "Add new file type";
+        }
+        
         if (dParent != null) {
-            diag = new JDialog(dParent, Localization.lang("Edit file type"), true);
+            diag = new JDialog(dParent, Localization.lang(title), true);
```

Comment:
Some lines below, the same strings are inlined. Therefore, a comment on the strings a few lines above is enough. (I hope to remember the setting correctly, currently on the road)

----- Ursprüngliche Nachricht -----
Von: "Simon Harrer" <notifications@github.com>
Gesendet: ‎05.‎10.‎2015 09:50
An: "JabRef/jabref" <jabref@noreply.github.com>
Cc: "Oliver Kopp" <koppdev@googlemail.com>
Betreff: Re: [jabref] Replaced DefaultFormBuilder with FormBuilder (#205)

In src/main/java/net/sf/jabref/external/ExternalFileTypeEntryEditor.java:
>          if (dParent != null) {
> -            diag = new JDialog(dParent, Localization.lang("Edit file type"), true);
> +            diag = new JDialog(dParent, Localization.lang(title), true);
We have a script that automatically extracts the used language keys. This only works when we do not pass a variable to the lang method. Can you please inline the language keys as Strings?
—
Reply to this email directly or view it on GitHub.